parquet-column/src/main/java/org/apache/parquet/column/ParquetProperties.java

```diff
@@ -56,6 +60,7 @@
   public static final int DEFAULT_COLUMN_INDEX_TRUNCATE_LENGTH = 64;
   public static final int DEFAULT_STATISTICS_TRUNCATE_LENGTH = Integer.MAX_VALUE;
   public static final int DEFAULT_PAGE_ROW_COUNT_LIMIT = 20_000;
+  public static final int DEFAULT_MAX_BLOOM_FILTER_BYTES = 1024 * 1024;
```

Comment:
@chenjunjiedada 
I'm curious about the maximum default value. Could you please explain why you choose 1 MB?

Reply:
Assume we have a row group with only one column of UUID (36 bytes), according to the formula and FPP = 0.01 we will need about 4MB. I expect that we will have more columns in the real scenario.

Reply:
@chenjunjiedada Thanks for the clarification!